common/source/docs/common-toshiba-can-escs.rst

```diff
@@ -47,7 +47,7 @@ Connection and Configuration
     :target: ../_images/toshiba-can-esc-dip-switches.png
     :width: 400px
 
-- Set the dip switches on each ESC to specify its unique number.  This number also maps the ESC to an equivalent PWM output on the back of the flight controller.  For example ESC 1 will react just like a normal PWM ESC connected to the flight controller's MAIN OUT 1.  By default this mapping also matches the :ref:`motor order diagrams <Copter:connect-escs-and-motors>`.
+- Set the dip switches on each ESC to specify its unique number.  This number also maps the ESC to an equivalent PWM output on the back of the flight controller.  For example ESC 1 will react just like a normal PWM ESC connected to the flight controller's MAIN OUT 1.  By default this mapping also matches the :ref:`motor order diagrams <Copter:connect-escs-and-motors3>`.
```

Comment:
I think the link should remain to connect-escs-and-motors instead of -motors3..

Reply:
For whatever reason, the original creates a build error....the fix does not and points to the correct location on the copter motors page....this same linking is used in several other places that navigate correctly....I dont know enough about Sphinx as to why the original fails, but the change works...must be that off site refs must use a full URL or the hyperlink on the pages subheader....it just works...